storybook/components/button.story.exs

```diff
@@ -0,0 +1,24 @@
+defmodule Storybook.Components.Button do
+  use PhxLiveStorybook.Story, :component
+
+  alias PhxLiveStorybook.Stories.Variation
+
+  # iframe needed to properly pickup font family.
+  def container, do: :iframe
```

Comment:
I asked about this on the forum but have no guidance yet.

https://elixirforum.com/t/what-is-the-best-way-to-get-phx-live-storybook-to-use-the-same-font-as-my-site-in-the-component-on-display/52687